### DISCUSSION TITLE
Load more - increasingly larger requests

### DISCUSSION THREAD

**Discussion by GioChocolateBro** [livewire/livewire#1225] [4. General]
I have a project going with a load more/infinite scroll. 
Logically every time the response with rendered HTML is getting larger and slower.

Ideally we only get the new items and append these to the feed VS getting and rendering all of them every request.
How would I go about this, or is there something to configure to achieve this?

This is what I have going right now:

```
    public $page = 0;
    public $limit = 20;
    public $total;

    public function render()
    {
        $query = ProgramItem::where('published', 1)
        $this->total = $query->count();
        $limit = ($this->page + 1) * $this->limit;
        $query = $query->limit($limit);
        return view('livewire.program-overview', [
            'programItems' => $query->get()
        ]);
    }

    public function loadMore()
    {
        $this->page++;
    }
```

I know I could add ->skip() so I can only get the new set, afterwards emit an event to javascript to append them. I wondered if there was a more "livewire" way to do this.

**Comment by amaelftah** [comment]
@GioChocolateBro i just faced the same situation . have you figured a way to solve it ?

**Comment by iksaku** [comment]
Livewire's main focus is to have a way to keep the backend and frontend in sync magically... At the moment, this seems like an expected behaviour, since requests that are going back and forth will carry the data from previous requests, in order to minimize hits to your Database.

The only workaround I've got for this kind of situations is to not use Livewire at all, because at the moment there is no way to "easily" detach certain parts of the request from both the backend and frontend diffing.

**Comment by Wulfheart** [comment]
Related: https://github.com/livewire/livewire/discussions/5363
